install.sh

```diff
@@ -9,6 +9,15 @@ migrate_from=""
 use_snapshots=1
 download_workers=256
 
+# config.json, genesis.json install url
+if [ "$network" = "testnet" ]; then
+  config_install_url="https://files.deploy.aurora.dev/testnet/config.json"
```

Comment:
@nujabes403 Hi! This is the old config, and if you try this link – it won't work right now.
The correct config should be at:
https://files.deploy.aurora.dev/testnet-new-rpc/config.json

Reply:
Thanks. @Karkunow